modules/gdscript/gdscript_compiler.cpp

```diff
@@ -267,7 +270,10 @@ GDScriptCodeGenerator::Address GDScriptCompiler::_parse_expression(CodeGen &code
 					// Perform getter.
 					GDScriptCodeGenerator::Address temp = codegen.add_temporary(codegen.script->static_variables_indices[identifier].data_type);
 					GDScriptCodeGenerator::Address class_addr(GDScriptCodeGenerator::Address::CLASS);
-					Vector<GDScriptCodeGenerator::Address> args; // No argument needed.
+					Vector<GDScriptCodeGenerator::Address> args;
+					if (codegen.script->member_indices[identifier].getter_size == 1) {
```

Comment:
This is getting the value from the wrong vector. Should use the static variables one:

```suggestion
					if (codegen.script->static_variables_indices[identifier].getter_size == 1) {
```

Reply:
Good catch! I amended my commit with the fix.